WooCommerce/Classes/ViewRelated/Orders/Order Creation/FlowCoordinator/NewSimplePaymentsNoticeViewModel.swift

```diff
@@ -0,0 +1,54 @@
+import SwiftUI
+
+final class NewSimplePaymentsNoticeViewModel {
+
+    private let simplePaymentsNoticeView: BottomSheetNoticeView
+
+    /// Redirects to `HubMenu`tabBar
+    ///
+    private var completionHandler: (() -> Void)? = {
+        guard let mainTabBarController = AppDelegate.shared.tabBarController else {
+            return
+        }
+        mainTabBarController.navigateTo(.hubMenu)
+    }
+
+    init() {
+        simplePaymentsNoticeView = BottomSheetNoticeView(
+            buttonTapped: completionHandler,
+            title: Localization.title,
+            message: Localization.message,
+            confirmationButtonMessage: Localization.confirmationButton
+        )
+    }
+
+    func setupNewSimplePaymentsNoticeView(for viewController: UIViewController) {
+        let hostingController = UIHostingController(rootView: simplePaymentsNoticeView)
```

Comment:
This is view related logic, that should not be in the view model actually. We can move this into the `UIViewController` :)

Reply:
Gotcha! Also thanks for the further explanation during our session, I moved the view-related logic to the ViewController and extracted it to its own file: e128e1e31d27efa2dee2138d6a1c4d584d245ee8